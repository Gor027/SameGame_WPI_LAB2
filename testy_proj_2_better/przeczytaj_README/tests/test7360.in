``UUUU``` `   U``U `` U` ` UUUU  ` `U`  `
U U   UU UU  ``UUU  UU`U` `` U ` U`  U`  UU`   `UU
U `   ` `` U` `  UUU`` U 
UUUUU`````` `U`  `  `UUUU UUU  UUU` 
U``U``U UUUU ` UU` `UUUU` `U U U `  `
`U  `    `  U`   ` UU `U``U`  U``` U U
