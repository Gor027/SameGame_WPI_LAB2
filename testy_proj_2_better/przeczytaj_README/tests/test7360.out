                 U  `  `
` U   `  U       U  U UU   U     `
U`U U U ```  `U  U ``U`U` ``   ` U
UU`UUU````U `U```` U``UUU UU U U U   `
U`UU```U`U`U```UU` `UUUU```UUUUUU```UU
UU`U``U`U`UUU``UU`UUUU`U``U`UUU````U`U``UU``UU
